src/pricing/ironswap.ts

```diff
@@ -0,0 +1,27 @@
+// pricing for ironswap LP tokens
+
+import { Address, BigInt, BigDecimal, log } from '@graphprotocol/graph-ts'
+import { IronSwapToken } from '../../generated/templates/GeyserV1/IronSwapToken'
+import { IronSwap } from '../../generated/templates/GeyserV1/IronSwap'
+import { integerToDecimal } from '../util/common'
+
+
+export function isIronSwapLiquidityToken(address: Address): boolean {
+  let token = IronSwapToken.bind(address);
+
+  let res = token.try_swap();
```

Comment:
Maybe I'm overthinking this, but I'm slightly worried this is a common enough method name that it may return true for non-ironswap tokens

Reply:
Okay, I actually had the same thought and disregarded. But the fact that you did too tells me it's worth addressing. Will push up a tweak to make this check stricter